iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/extractor/dataregion/realtime/assigner/PipeDataRegionAssigner.java

```diff
@@ -218,18 +241,24 @@ public boolean notMoreExtractorNeededToBeAssigned() {
   public void close() {
     PipeAssignerMetrics.getInstance().deregister(dataRegionId);
     matcher.clear();
-    disruptor.clear();
+
+    final long startTime = System.currentTimeMillis();
+    disruptor.shutdown();
+    LOGGER.info(
+        "Pipe: Assigner on data region {} shutdown internal disruptor within {} ms",
```

Comment:
```suggestion
        "Pipe: Assigner on data region {} shutdown internal disruptor within {} ms",
```

Please include message of "pipe name - creation time - region id" in logger's output

Reply:
could not fetch such info since per assigner per region?